react/src/lib/components/DeckGLMap/layers/LassoLayer/lassoLayer.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import { makeStyles } from "@material-ui/core";
+import Switch from "@material-ui/core/Switch";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import DeckGLMap from "../../DeckGLMap";
+
+export default {
+    component: DeckGLMap,
+    title: "DeckGLMap / Lasso Layer",
+} as ComponentMeta<typeof DeckGLMap>;
+
+const useStyles = makeStyles({
+    main: {
+        height: 500,
+        border: "1px solid black",
+        position: "relative",
+    },
+    legend: {
+        width: 100,
+        position: "absolute",
+        top: "0",
+        right: "0",
+    },
+});
+
+export const lassoLayerTemplate: ComponentStory<typeof DeckGLMap> = (args) => {
```

Comment:
Please change the name of the story into something more descriptive, eg. "LassoSelection". This is not really a template.